### ISSUE TITLE
Add variable variable variables

### ISSUE THREAD

**Issue by flexagoon** [TodePond/DreamBerd#144]
With `const const const`s being in DreamBerd, I think `var var var`s would also be a welcome addition. Variable variable variables are variables that can take any value at any time. This would allow programmers to write more efficient code when a value needs to change really often.

```
var var var name = "Lu"!
print(name == "Lu")! // True
print(name == "Hotdog")! // True
print(name == 3)! // True
print(name)! // [7, 4, 8]
```

**Comment by TodePond** [comment]
Good idea. Wanna do a PR? This seems like quite an advanced feature so could go quite far down the page

**Comment by flexagoon** [comment]
> Good idea. Wanna do a PR? This seems like quite an advanced feature so could go quite far down the page

Sure, I'll work on it when I have the time

**Comment by GoldenStack** [comment]
You could also consider making the `var var var` so variable that even its name is variable, so all declared `var var var` vars hold the same value.
